assets/src/edit-story/components/library/libraryTabs.js

```diff
@@ -68,6 +68,7 @@ function LibraryTabs() {
         <Tab
           key={id}
           id={getTabId(id)}
+          paneId={getPaneId(id)}
```

Comment:
Why not just `aria-controls` here? Why go via `.attrs()`?

Reply:
Noticed that it's actually passed in to the tab instance so it's not necessary here